portfolio/src/main/webapp/index.html

```diff
@@ -76,19 +76,16 @@ <h3 class="content-title" id="facts-title">Random Facts</h3>
     </article>
   </div>
   <div id="comment-container">
-    <form action="/data" method="post">
-      <p>Add a comment!
-      <input type="text" name="comment" value="comment">
-      <input class="submit-button" type="submit" value="Post Comment">
-    </form>
+    <h3>Comments!</h3>
     <form>
       <p>Show how many comments?
       <input id="comment-amount" min="1" max="50" name="comment-amount" type="number" value="10">
     </form>
-    <button class="submit-button" id="delete-comments">Delete Comments</button>
+    <button id="delete-comments">Delete Comments</button> 
+    <button id="to-comment-page">Post a Comment</button>
     <hr>
     <div id="comments"></div>
   </div>
 </div>
-<canvas id="canvas" width=2000 height=230></canvas>
+<canvas id="canvas" width=500 height=100></canvas>
```

Comment:
I haven't done anything with the canvas yet, but I want to add an animation at the bottom of the screen when I have time to.